feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalController.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.esri.samples.feature_collection_layer_from_portal;
+
+import com.esri.arcgisruntime.data.FeatureCollection;
+import com.esri.arcgisruntime.layers.FeatureCollectionLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+import javafx.fxml.FXML;
+import javafx.scene.control.Alert;
+import javafx.scene.control.TextField;
+
+public class FeatureCollectionLayerFromPortalController {
+
+    @FXML
+    private MapView mapView;
+    @FXML
+    private TextField FeatureCollectionItemIdTextField;
+    private ArcGISMap map;
+
+    public void initialize() {
+        try {
+
+            // create amp and set it to be displayed in this view
+            map = new ArcGISMap(Basemap.createOceans());
+            mapView.setMap(map);
+            FeatureCollectionItemIdTextField.setText("32798dfad17942858d5eef82ee802f0b");
+            fetchFromPortal();
+
+
+        } catch (Exception e) {
+            // on any error, display the stack trace.
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Handles searching the provided Item ID.
+     */
+    @FXML
+    private void fetchFromPortal() {
```

Comment:
Nice work on figuring out FXML and the Controller setup!
I do think that the UI here is quite small, since it's just a text field and one button. So we can easily put it all just in the `FeatureCollectionLayerFromPortalSample.java' file and do everything from there. 
Check out the samples 'Change Feature Layer Renderer' and 'OpenMapURL', they are quite simple and somewhat similar. I think you could do it in a similar way, just build a small UI in the main Sample class. Give me a shout if you'd like any advice!

Reply:
as a best practice can we still keep MVC pattern here for this sample ?